### ISSUE TITLE
Source files of completed conversions in queue kept in use by Finder until quit.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [XeresRazor/subler#455]
```
What steps will reproduce the problem?
1. Queue multiple conversions of .mkv to .m4v files.
2. Delete the source (mkv) files of files that are finished in the queue.
3. Empty the trash in finder.

What is the expected output? What do you see instead?
Expected output would be that the files will be trashed.
Instead there will be a notification stating these files are still in use 
(The operation can’t be completed because the item “.......mkv” is in 
use.)

What version of the product are you using? On what operating system?
Latest (0.17), OSX Mountain Lion. This is not a new issue, I noticed the same 
on 0.16.

Please provide any additional information below.
Simply put I would like to be able to trash the source files after conversion, 
even before the total queue is done. Now I have to wait until the entire queue 
is done. Sometimes I just don't have enough space for an entire queue of 
originals and converted files...
```

Original issue reported on code.google.com by `sjo...@gmail.com` on 17 Aug 2012 at 7:32
- Merged into: #429
